static/app/views/starfish/views/spanSummaryPage/index.tsx

```diff
@@ -65,7 +65,7 @@ function SpanSummaryPage({params, location}: Props) {
   const {groupId} = params;
   const {transaction, transactionMethod, endpoint, endpointMethod} = location.query;
 
-  const {data: fullSpanDescription} = useFullSpanDescription(groupId);
+  const {data: fullSpan} = useFullSpanFromTrace(groupId);
```

Comment:
should we add loading state here? (we can add in follow up PR)

Reply:
Great Q, I was torn on this. When this page loads, a partial query is already pre-populated. I didn't want to block it with a loading spinner while we fetch the full query because even the short query is valuable, so I went without. Maybe some kind of small spinner would be good though, eh?

Reply:
I think the smaller spinner is a good idea, but it shouldn't be a blocker

Reply:
Roger, I'll bring this up with Dora!